src/Venturecraft/Revisionable/Revision.php

```diff
@@ -187,6 +187,22 @@ public function userResponsible()
         }
     }
 
+
+    /**
+     * Returns the object we have the history of
+     * @return Object or false
+     */
+    public function historyOf()
+    {
+        if(class_exists($class = $this->revisionable_type))
+        {
+            $dd = $class::find($this->revisionable_id);
+            dd($dd);
```

Comment:
Sorry, made that change local, but forgot to push it to github. should be good to go now.